src/index.html

```diff
@@ -11,7 +11,191 @@
+            id="birthDate"
+            name="birthDate"
+            required
+          >
+        </div>
+
+        <div class="form__element">
+          <label for="agreement">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="agreement"
+            name="agreement"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form__group">
+        <legend>Registration</legend>
+        <div class="form__element">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="form__element">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            minlength="6"
+            maxlength="20"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form__group">
+        <legend>An interesting fact about you!</legend>
+        <div class="form__element">
+          <label>Do you love cats?</label>
+
+          <label>
+            <input
+            type="radio"
+            name="loveCats"
+            value="yes"
+            >
+              Yes
+          </label>
+
+          <label>
+            <input
+            type="radio"
+            name="loveCats"
+            value="no"
+            >
+              No
+          </label>
+        </div>
+
+        <div class="form__element">
+          <label for="favoriteColor">What is your favorite color?</label>
+          <input
+            type="color"
+            id="favoriteColor"
+            name="favoriteColor"
+          >
+        </div>
+
+        <div class="form__element">
+          <label for="goToBed">What time do you go to bed?</label>
+          <input
+            type="time"
+            id="goToBed"
+            name="goToBed"
+          >
+        </div>
+
+        <div class="form__element">
+          <label for="favoriteCars">What are your favorite brand of cars?</label>
+          <select
+            name="favoriteCars"
+            id="favoriteCars"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+
+        <div class="form__element">
+          <label for="rate">How do you rate our work?</label>
+          <input
+            type="range"
+            id="rate"
+            name="rate"
+            min="0"
+            max="10"
+            value="0"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form__group">
+        <legend>Additional info:</legend>
+        <div class="form__element">
+          <label for="coments">Comments:</label>
```

Comment:
checklist p.8 - Check the textarea behaviour - when you click on the textarea, the coursor should appear on the beginning of the input.

fix it

Reply:
Today in class I asked Sasha Yatsykovych about this error and this error is related to the linter does not allow the opening and closing tags to be on the same line